main.go

```diff
@@ -33,8 +33,16 @@ type input struct {
 func main() {
 
 	var pulsepid int
+	var sourceName string
+	var unload bool
+	var threshold int
+
 	flag.IntVar(&pulsepid, "removerlimit", -1, "for internal use only")
+	flag.StringVar(&sourceName, "s", "", "Load PulseAudio source by name")
```

Comment:
How does the user find the source name? We should have a flag (`-l` for list, for example) that prints inputs and makes it clear to the user what the source name is and how to supply it, without users having to use `pacmd` or similar to get this information.

Reply:
I think this is a good idea. I've added a commit that implements this listing functionality. With that said, how would we feel about adding a verbose flag, to prevent debug logs from being printed to stdout during normal use? As of now, stdout gets pretty busy.

Reply:
I think the easiest would be to just stop logging to stdout, now that we actually use it and only log to the file. See here: https://github.com/lawl/NoiseTorch/blob/master/main.go#L54

Reply:
Ok, I removed the multiwriter, and enforced stderr output for relevant errors for console output